### ISSUE TITLE
Azure AD is now Microsoft Entra

### ISSUE THREAD

**Issue by ecfan** [SharePoint/sp-dev-docs#9564]
Here's the supporting links:

-	[Microsoft Entra - Secure Identities and Access | Microsoft Security](https://www.microsoft.com/en-us/security/business/microsoft-entra)
-	[Microsoft Entra ID (formerly Azure Active Directory) | Microsoft Security](https://www.microsoft.com/en-us/security/business/identity-access/microsoft-entra-id)
-	[Microsoft Entra ID documentation - Microsoft Entra ID | Microsoft Learn](https://learn.microsoft.com/en-us/entra/identity/)

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: e49298d0-77a4-349a-f7ad-75b9f5773d33
* Version Independent ID: e49298d0-77a4-349a-f7ad-75b9f5773d33
* Content: [Understanding Azure Active Directory and OAuth 2.0 in the context of SharePoint Online modern development](https://learn.microsoft.com/en-us/sharepoint/dev/sp-add-ins-modernize/understanding-aad-and-oauth-for-spo-modern)
* Content Source: [docs/sp-add-ins-modernize/understanding-aad-and-oauth-for-spo-modern.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/sp-add-ins-modernize/understanding-aad-and-oauth-for-spo-modern.md)
* Service: **sharepoint**
* GitHub Login: @VesaJuvonen
* Microsoft Alias: **vesaj**

**Comment by Ashlesha-MSFT** [comment]
Hello @ecfan,
Thank you for bringing this issue to our attention. We will look into it and get back to you shortly.

**Comment by Ashlesha-MSFT** [comment]
@ecfan,
You're right — Azure Active Directory is now Microsoft Entra ID. We'll connect with the engineering team to review and update the documentation accordingly to reflect the new naming and branding.

Appreciate you bringing this to our attention and sharing the supporting links.

**Comment by ecfan** [comment]
[like]  Esther Fan reacted to your message:
________________________________
